packages/components/credentials/ElectricsearchUserPassword.credential.ts

```diff
@@ -15,7 +15,7 @@ class ElasticSearchUserPassword implements INodeCredential {
             'Refer to <a target="_blank" href="https://www.elastic.co/guide/en/kibana/current/tutorial-secure-access-to-kibana.html">official guide</a> on how to get User Password from ElasticSearch'
         this.inputs = [
             {
-                label: 'Cloud ID',
+                label: 'Cloud ID or custom server URL',
                 name: 'cloudId',
```

Comment:
can we add a `description` to explain how to get Cloud ID OR what is custom server URL? Better if it has link to docs like [here](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/credentials/AWSCredential.credential.ts#L16)